templates/birds/utility.macros.j2

```diff
@@ -31,6 +31,24 @@
 {% endmacro %}
 
 
+{% macro get_og_author(author) -%}
+  {% set page = all_pages|filter_pages('bird', 'eq', author)|get_page %}
+  {%- if page and page.config.facebook -%}
+    {{ ['https://www.facebook.com/', page.config.facebook]|join('') }}
```

Comment:
If we put `facebook` under the `social` key, this would then be `page.config.social[0].facebook`, which is what we've got elsewhere for social media things.